ampersand-chart.js

```diff
@@ -467,9 +473,11 @@
           }
         }]
       });
-      filterTrackerState.onApply = function(props) { 
+      filterTrackerState.onApply = function(props, options) { 
         this.model.filterOnApply(props);
-        this.toggleFilterWindow();
+        if (options.doNotToggle !== true) {
```

Comment:
It's actually mostly undefined and I don't like trusting JS's type coercion. /giphy paranoia